docs/workshop/onboarding.md

```diff
@@ -1,3 +1,74 @@
 # Onboarding
 
-This section will consist of introduction to the workshop for the attendees and consist of onboarding and login details for the operate first environment (tools like Jupyterhub, Openshift console, Trino, Cloudbeaver, S3 bucket, Superset).
+Hello and welcome to the workshop on building a simple AIOps tool on the [Operate First cloud](https://www.operate-first.cloud/) and learning how to move your AI workloads to the cloud by implementing an end-to-end ML tool.
+
+In this workshop, you will learn how to collect data from an open CI/CD data source, use Jupyterlab and its extensions to manage dependencies, and run jupyter notebooks. You will learn how to perform automated, repeatable experiments using Elyra and Kubeflow Pipelines, create and deploy machine learning modes on OpenShift using Seldon and visualize results on a Superset dashboard.
+
+To get started with this tour, let’s first introduce you to the various tools that are available on Operate First environment and will help us in building the AIOps application using Operate First.
+
+
+
+1. **Openshift Console -**
+
+The Operate First initiative currently manages two clusters within the [MOC environment](https://massopen.cloud/).
+
+* [Smaug cluster](https://console-openshift-console.apps.smaug.na.operate-first.cloud/) for all user workloads
+* [Infra cluster](https://console-openshift-console.apps.moc-infra.massopen.cloud/) for cluster management, housing ACM and ArgoCD
+
+For this workshop, we shall proceed with using the [Smaug cluster](https://console-openshift-console.apps.smaug.na.operate-first.cloud/).
+
+![Cluster Login Page](../assets/images/cluster1.png)
+
+Select login with `operate-first`, and login into the cluster by using your Github credentials.
+
+![Cluster Landing Page](../assets/images/cluster2.png)
+
+2. **Ceph S3 storage -**
+
+Ceph provides a unified storage service with an object, block, and file interface from a single cluster built from commodity hardware components. Operate First hosts a public ceph storage bucket that is accessible by anyone. You can find the credentials and other information for this bucket below:
```

Comment:
We need to update these bucket creds to the new bucket. We need to check with Humair if the creds https://console-openshift-console.apps.smaug.na.operate-first.cloud/k8s/ns/aiops-tools-workshop/secrets/s3buckets should be available as markdown here? Might be better to link to the credentials hosted in a text editor for the workshop

Reply:
Yes, I was waiting on the testing bit before we add those details here. And regarding the creds being available here, this is how operate first gitops documentation has presented these [here](https://www.operate-first.cloud/apps/content/odh/trino/access_public_bucket.html). Other option would be vault or any other text editor we decide on